PlayerInfo/PlayerInfo.cpp

```diff
@@ -130,5 +174,217 @@ namespace Plugin {
         }
     }
 
+    uint32_t PlayerInfo::getSinkAtmosCapability (const JsonObject& parameters, JsonObject& response)
```

Comment:
As already explained in our Development Perception, we believe in Strong typed development. The compiler is our friend. We love the compiler to tell us when we are making mistakes, hence why we propose to use the JSON contract definitions and the using these strong type generated JSON onject classes. This way, errors are already discovered during compile time (not Intergration time) which safes a lot of development time.
Also the generated classes will avoid the large amount of magicals, like: "atmos_capability" in the code.